04.ls/ls.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+SPLIT_NUMBER = 3
+
+def group_files
+  obtained_files = Dir.glob('*')
+  max_length = (obtained_files.length.to_f / SPLIT_NUMBER).ceil
+
+  grouped_files = obtained_files.each_slice(max_length).to_a
+  blank_numbers = grouped_files[0].length - grouped_files[-1].length
+  grouped_files[-1] += Array.new(blank_numbers, nil)
+  grouped_files
+end
+
+def output_like_ls_command(completed_grouped_files)
```

Comment:
```suggestion
def output_like_ls_command(grouped_files)
```

completedは無くてもいい気がしますね

Reply:
@JunichiIto 
かしこまりました🙇